### DISCUSSION TITLE
Any plans for built in support of GitHub Codespaces in iPad/tablet version?

### DISCUSSION THREAD

**Discussion by Anselmoo** [community/community#135358] [Mobile] [answered]
### Select Topic Area

Question

### Body

It would be fantastic if GitHub's mobile app for iPad supported the direct opening of GitHub Codespaces in the web version within the app, eliminating the need to switch to a mobile browser. 

I think this would work well for minor bug fixes or code checks.

**Comment by taki-on** [comment]
@Anselmoo Thank you for your suggestion. We have been discussing the best way to support Codespaces on iPad but we do not have any concrete plan for that yet. However, it is definitely on our list!

**Reply by Anselmoo** [reply, marked as answer]
thx for quick response @taki-on

**Reply by Anselmoo** [reply]
https://blink.sh enables VSCode Codespaces on mobile devices without Safari restricting the usable window size.
